### ISSUE TITLE
Login loop on internal network using Firefox / Chrome

### ISSUE THREAD

**Issue by rp-tech** [osTicket/osTicket#1757]
I am having a very strange issue with OSTicket.

When I try to log to the client side using AD authentication and Firefox / Chrome from another host on the SAME network as my OSTicket server, it simply loops back to the login screen.

Using IE works fine.

If I try to log in using a computer OUTSIDE my network, the login works perfectly using any web browser.

I am completely stumped as to what could be the cause - all my other web apps on the same box work fine, using any browser on any host on any network (outside or inside).

Also, SCP works perfectly fine using any browser, no matter where the client is located.

We have our DNS set up as follows:

Outside A record -> public IP -> port forward 443 to private IP of OSticket webserver (using SSL)
Inside A record on Windows DNS server -> private IP of OSticket webserver

System info:

![2015-02-20 12_03_07-osticket __ admin control panel](https://cloud.githubusercontent.com/assets/10659713/6308363/a0b2371a-b8f8-11e4-9aac-fa546a1a33c2.png)

Any help with this issue would be appreciated.

**Comment by rp-tech** [comment]
Just FYI I tried absolutely everything to resolve this issue.

I had it working using a clean install of Debian inside a VirtualBox guest VM.

Moved the VM to my Hyper-V server and it started doing the same thing!

After re-installing EVERYTHING, I couldn't figure out what was going on - so I tried dumping Apache in favor of NGINX - which works!

So there is some weird issue happening with OsTicket / Apache specifically on my Hyper-V Debian guest.

But all my other Debian / Apache software works fine! (MediaWiki, Observium, etc).

Any ideas why Apache would not work properly with Firefox / Chrome on the internal network specifically when running under Hyper-V - a bizarre issue for sure.

Also tried turning off Hyper-V integration services + using a legacy adapter - no luck!

**Comment by rp-tech** [comment]
Last follow-up - I migrated to a CentOS LAMP stack and everything works fine.

Upgraded Apache on the Debian box to the same version as on CentOS - fixed the issue in Firefox but not in Chrome. So in the end I ended up switching everything over to CentOS as it seems to be way more stable.

So I don't think this is an OSTicket issue specifically, rather the combination of running Debian Wheezy, Apache 2.2, and OSTicket inside a Hyper-V guest (note the last part, as everything works flawlessly outside of Hyper-V).
